### DISCUSSION TITLE
Grab tag info from a URL?

### DISCUSSION THREAD

**Discussion by nmycps** [Marekkon5/onetagger#276] [Ideas]
When my CD ripper app cannot find a source that matches a CD on its own, it will prompt for a URL that can be entered where the app can identify the CD correctly. Wondered if that would be a useful feature for onetagger users?

**Comment by Marekkon5** [comment]
Hello this is quite complicated because OneTagger has no interface to search for tracks by URL/ID, only some platforms support specific custom tags. Some platforms also rely on APIs and some on release URL, so just passing an URL wouldn't work in most cases.

However Manual Tag should be good enough if you wanna cherry pick your matches.
